README.md

```diff
@@ -117,6 +117,17 @@ Take the following steps to set up Juicebox's subgraph for local development:
    yarn start
    ```
 
+### Testing
+
+** Make sure you start the app using `yarn start` first **
```

Comment:
**praise** thanks for adding this to the docs.

**suggestion** maybe we can add 2 subheadings:

- **Unit tests**
- **End-to-end tests**

And put the below under **End-to-end tests**, and add a small guide about running the unit tests (`yarn test`)?

Reply:
Yeah absolutely should add this thanks!

Reply:
Diff from pairing today:

```diff
diff --git a/.example.env b/.example.env
index 9e3812d0..2885b8a6 100644
--- a/.example.env
+++ b/.example.env
@@ -21,8 +21,8 @@ REACT_APP_SENTRY_DSN=
 
 # Secret phrase used for e2e testing with metamask.
 # !!-- ** USE A TEST WALLET, NOT YOUR REAL WALLET!!!! ** -- !! 
-E2E_SECRET_WORDS=
+E2E_SECRET_WORDS="syrup front comfort vapor sunny project impose tired potato harvest margin extend"
 # Password used for e2e testing with metamask. Set this to whatever you prefer.
-E2E_PASSWORD=
+E2E_PASSWORD=12345678
 # Name of the network to use for E2E testing
 E2E_NETWORK=rinkeby
\ No newline at end of file
diff --git a/README.md b/README.md
index 8ff2ffc0..59faf8e2 100644
--- a/README.md
+++ b/README.md
@@ -105,8 +105,9 @@ using a GraphQL API.
 Take the following steps to set up Juicebox's subgraph for local development:
 
 1. Join [Peel's discord server](https://discord.gg/akpxJZ5HKR).
-2. Go to [the dev channel](https://discord.com/channels/939317843059679252/939705688563810304)
-    and inquire about mainnet and rinkeby subgraph URLs.
+2. Go to
+   [the dev channel](https://discord.com/channels/939317843059679252/939705688563810304)
+   and inquire about mainnet and rinkeby subgraph URLs.
 3. Copy the URL into the `REACT_APP_SUBGRAPH_URL` variable of the `.env` file.
 
 ### Usage
@@ -117,16 +118,43 @@ Take the following steps to set up Juicebox's subgraph for local development:
    yarn start
    ```
 
-### Testing
+## Testing
 
-** Make sure you start the app using `yarn start` first **
+### Unit tests
+
+Run the jest test suite using the following command:
+
+```bash
+yarn test
+```
+
+### End-to-end tests
+
+#### Configuration
+
+Setup your .env file with the `E2E_` prefixed variables.
+
+- `E2E_SECRET_WORDS`: 12 word Mnemonic. Generate this here:
+  https://iancoleman.io/bip39/. **Important**: wrap mnemonic in quotes (`"`).
+- `E2E_PASSWORD`: Any random password with at least 8 characters. Cypress will use this to create the fake
+  MetaMask account.
+- `E2E_NETWORK`: Set this to `rinkeby` (unless you have a reason not to).
+
+#### Running Cypress
+
+1. Start the app in a seperate terminal:
+
+   ```bash
+   yarn start
+   ```
+
+1. Open Cypress.
 
-1. Setup your .env file with the `E2E_` prefixed variables.
-2. Open Cypress.
    ```bash
    yarn cy:open
    ```
-3. Run tests.
+
+1. Run tests.
 
 ## Contributing
 
```